base/system/services/rpcserver.c

```diff
@@ -1686,6 +1698,8 @@ RSetServiceStatus(
     LPCWSTR lpLogStrings[2];
     WCHAR szLogBuffer[80];
     UINT uID;
+    HANDLE hStopThread;
+    DWORD dwStopThreadId;
```

Comment:
Suggestion: move these two variables inside the if-block that calls the CreateThread() function below.

Reply:
Moved. Thanks.